tests/test_config.py

```diff
@@ -252,40 +252,54 @@ def test_load_target_exception(self):
         oca: https://github.com/OCA/product-attribute.git
     merges:
         - oca 8.0
+    target: oca 8.0 extra_arg
 """
         with self.assertRaises(ConfigException) as ex:
             config.get_repos(self._parse_config(config_yaml))
-        self.assertEquals(ex.exception.args[0],
-                          '/product_attribute: No target defined.')
+        self.assertEquals(
+            ex.exception.args[0],
+            '/product_attribute: Target must be formatted as '
+            '"[remote_name] branch_name"')
 
         config_yaml = """
 /product_attribute:
     remotes:
         oca: https://github.com/OCA/product-attribute.git
     merges:
         - oca 8.0
-    target:
+    target: oba aggregated_branch
```

Comment:
Can we keep the existing tests and add new ones for the new use cases ?

Reply:
That existing test was testing things you weren't allowed to do:
- Missing `target`
- Empty `target`
- Bad remote in `target`

The first two are allowed now, so I replaced them by an over-full `target` (which is still not allowed). I kept the third and added a test for the new use cases.

Reply:
I may misread, but then I don't see the test with Missing target and Empty target.

Reply:
Those are allowed now so the tests had to be removed/replaced. The test tested the error, but they no longer give an error.

I did add a new different test for a missing target. (An empty target is treated the same (`repo_data.get('target', "")`) so I didn't test that separately.)

Reply:
> Those are allowed now so the tests had to be removed/replaced.

I don't understand that. If this becomes allowed, the test can stay with the same input, but updated to test the expected outcome.

Reply:
`test_target_defaults` tests the same input with a new expected outcome. It didn't make sense to keep it in a test for testing exceptions, so I moved it.

Because a missing and empty `target` are so similar I only test a missing `target`, but I could add another test for an empty `target` if you prefer.